products/pluto/tests/assert_bbl_count_is_the_same.sql

```diff
@@ -0,0 +1,34 @@
+{{ 
+    config(
+        tags = ['de_check', 'minor'], 
+        meta = {
+            'description': '''
+				This test checks that final PLUTO table has same row count as previous version PLUTO. 
+                This test is expected to pass for PLUTO Minor versions only.
+			''',
+            'next_steps': '''
+				Investigate if a correct recipe file was used. If so, review stats for input datasets. 
+                This needs to be addressed prior to promoting PLUTO for GIS review.
+			'''
+        }
+    ) 
+}}
+
+WITH current_pluto AS (
+    SELECT bbl::bigint, 'current' AS source
+    FROM {{ source('build_sources', 'export_pluto') }}
+),
+previous_pluto AS (
+    SELECT bbl::decimal::bigint, 'previous' AS source
```

Comment:
Curious about this cast to decimal here. What necessitates that?

Reply:
Previous Pluto table has bbl as a decimal  text 🙃

Reply:
Maybe something to work on when we move the suite of `pluto` templates over to ingest